index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+    <title>Ruri's portfolio</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <h1>Ruri Enciso</h1>
```

Comment:
- According to [this requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Your%20website%20should%20look%20exactly%20like%20the%20design%20(e.g.%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.), Your website should look exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). I think it would be better to fix that and stick to the Figma template.

**Original Design**
![Screenshot from 2023-04-28 23-57-21](https://user-images.githubusercontent.com/97834160/235261813-9873b1d6-0208-4688-9ef1-3c00bac0b633.png)

**Your Design**
![Screenshot from 2023-04-28 23-57-10](https://user-images.githubusercontent.com/97834160/235261831-ee83f3aa-6b43-49dd-9287-ca80cc81728e.png)